pkgs/tools/virtualization/tart/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation rec {
```

No need to use a compiler if nothing is built.